src/PresentationalComponents/Input/Input.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+
+const checkTypes = ['checkbox', 'radio'];
+
+const Input = ({type, className, ...props}) => {
+  const classes = checkTypes.hasOwnProperty(type) ? 'pf-c-check' : 'pf-c-form-control';
+  return (
+    <input {...props}
+      type={type}
+      className={classnames(classes, className)}
+    />
+  );
+};
+
+Input.propTypes = {
+  type: PropTypes.string,
+  className: PropTypes.string
+}
```

Comment:
Are we going to be adding the ability to have disabled inputs?

Reply:
Yes, if you look at [props spread](https://github.com/RedHatInsights/insights-frontend-components/pull/26/files#diff-374245968821659edef31240ab6fd102R10) disabled prop will be passed into input.